<!DOCTYPE html>
<!--
    Plain-Academic by Vasilios Mavroudis
    Released under the Simplified BSD License/FreeBSD (2-clause) License.
    https://github.com/mavroudisv/plain-academic
-->

<html lang="en">
<head>
  <title>Constantin Pape - Computational Cell Analytics</title>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.0/jquery.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
  <link href='https://fonts.googleapis.com/css?family=Oswald:700' rel='stylesheet' type='text/css'>
</head>
<body>


<!-- Navigation -->
	<nav class="navbar navbar-inverse navbar-static-top" role="navigation">
	  <div class="container">
		<div class="navbar-header">
		  <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
						<span class="sr-only">Toggle navigation</span>
						<span class="icon-bar"></span>
						<span class="icon-bar"></span>
						<span class="icon-bar"></span>
					</button>
		</div>

		<!-- Collect the nav links, forms, and other content for toggling -->
		<div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
		  <ul class="nav navbar-nav">
				  <li><a href="index.html">Home</a></li>
				  <li><a href="#research">Research</a></li>
				  <li><a href="#teaching">Teaching</a></li>
				  <li><a href="#publications">Publications</a></li> 
				  <li><a href="#cv">CV</a></li> 
		  </ul>
		</div>
	  </div>
	</nav>
  
  <!-- Page Content -->
    <div class="container">

        <div class="row">

            <!-- Entries Column -->
            <div class="col-md-8" style="height: 25vh;">
                
                <div style="margin-top:3%; text-align:justify;">                
                    <img class="img-responsive" src="img/constantin_pape.jpg" alt="" width="150" height="225" align="left"><br>
                    <p>I am a Juniorprofessor at Georg-August University Goettingen and a member of the <a href="https://uni-goettingen.de/en/619480.html">Institute of Computer Science</a> and <a href="https://www.uni-goettingen.de/en/634473.html">CIDAS</a>. I lead the research group for <b>Computational Cell Analytics</b>. My research is focused on deep learning methods for computer vision, in particular segmentation, applied primarily to microscopy images.</p>

                    <p>I have started my group in March 2022 and am looking for a PhD student, job ad will come soon!</p>

                    <p>Previously, I have worked as a PostDoc at EMBL Heidelberg in the group of Anna Kreshuk and did my PhD at the University of Heidelberg under the supervision of Anna Kreshuk and Fred Hamprecht. During my PhD and PostDoc I have mainly worked on instance segmentation problems, with a focus on large volumetric electron microscopy data.</p>
                </div>
            </div> 

            <!-- Contact Info on the Sidebar -->
            <div class="col-md-4">
                <div style="font-family: 'Oswald', sans-serif; font-size: 32px;"><b>Constantin Pape</b></div><br>
                <p><b>constantin.pape@informatik.uni-goettingen.de</b><br>
                <p>Institute for Computer Science<br>
                University of Goettingen<br>
                <!--
                Street<br>
                City <br>
                Country<br>
                </p>
                -->
            </div>
            
            
            <!-- Links on the Sidebar -->
            <div class="col-md-4" style="margin-top:2%">
              <dd><a href="https://scholar.google.com/citations?user=idkzOIUAAAAJ&hl=en">Google Scholar</a></dd> 
              <dd><a href="https://github.com/constantinpape">Github</a></dd> 
              <dd><a href="https://twitter.com/cppape">Twitter</a></dd>
            </div>


            <!-- Research -->
            <div class="col-md-8" style="height: 70vh;">    
                <h2 id="research">Research</h2>
                <p>My main research interest is in high-quality segmentation methods for microscopy that require minimal supervision.
                The adoption of deep learning has improved the quality of image analysis for microscopy dramatically in the last decade.
                In the same period, the throughput, field of view and time resolution of microscopes has also increaded significantly, requiring the automation of key analysis steps, such as the identification of cells or other structures of interest through segmentation.
                While deep-learning based methods yield high enough quality to achieve this automation in many instances, they require large amounts of (manually) annotated training data .
                This limitation makes them impractical for many applications where no such training data is available and is too costly to create.
                Fortunately, the last years have seen a large reseach interest in weak supervision, domain adaptation and self-supervision, which promises to make training of high-quality models with significantly less annotated data possible.
                However, these methods are mostly developed for natural images and their adoption for microscopy has so far been limited.
                I aim to bridge this gap, building on initial work for <a href="https://doi.org/10.3389/fcomp.2022.805166">domain adaptation</a>, <a href="https://arxiv.org/abs/2103.14572">weak supervision</a> and even <a href="https://arxiv.org/abs/2107.02600">reinforcement learning</a>.</p>

                <img class="img-responsive" src="img/fig.jpg" alt="" width="350" height="350" align="right"><br>
                <p>My previous research has been focused on boundary based instance segmentation. I have developed graph-based methods, using <a href="https://doi.org/10.1038/nmeth.4151">graph partitioning</a> and <a href="https://openaccess.thecvf.com/content_ECCV_2018/html/Steffen_Wolf_The_Mutex_Watershed_ECCV_2018_paper.html">fast heuristics</a>.
                The initial focus of my work has been neuron segmentation in electron micrscopy (D), for which I have developed <a href="https://openaccess.thecvf.com/content_ICCV_2017_workshops/w1/html/Pape_Solving_Large_Multicut_ICCV_2017_paper.html">methods that scale to multi terayte volumes</a> and <a href="https://doi.org/10.3389/fcomp.2019.00006">can incorporate biological priors</a>.
                These methods have been used in various life science applications: <a href="https://doi.org/10.1016/j.cell.2021.07.017">building a high resolution genetic and morphological atlas of P. dumerilii (A)</a >, <a href="https://doi.org/10.1126/science.abj2949">analyzing the morphology of precursor neural cells in sponges (B)</a > or <a href="https://doi.org/10.1002/bies.202000257">developing an imaging based SARS-CoV-2 antibody assay (C)</a >. I plan to continue this work
                and make these methods available through easy-to-use tools in order to democratize the
                access to large-scale volumetric segmentation.</p>

                <p>I am dedicated to open source and open science and am actively contributing to several related efforts. In particular, the <a href="https://bioimage.io/#/">bioimage.io modelzoo</a>, a resource to share deep learning models for microscopy image analysis and <a href="https://doi.org/10.1038/s41592-021-01326-w">ome.ngff</a>, a new image data format that supports efficient storage of large data and on-demand access in the cloud. Furthermore, I co-develop <a
                    href="https://mobie.github.io/">MoBIE</a>, a Fiji plugin for exploring and sharing large multi-modal image data.</p>
            </div>
            

            <!-- Teaching -->
            <div class="col-md-8" style="height: 15vh;">    
                <h2 id="teaching">Teaching</h2>
                I give a lecture on deep learning for computer vision and hold seminars on advanced topics in deep learning and machine learning. Please check <a href="http://cs.ugoe.de/pape">UniVZ</a> for my current courses.
            </div>
            
            
            <!-- Publications -->
            <div class="col-md-8" style="height: 40vh;">    
                <h2 id="publications">Publications</h2>
                
                Please find my full list of publications on <a href="https://scholar.google.com/citations?user=idkzOIUAAAAJ&hl=en">Google Scholar</a>. Below is a list of selected publications:
                <ul>
                <li class="paper"><a href="https://doi.org/10.3389/fcomp.2022.805166">From Shallow to Deep: Exploiting Feature-Based Classifiers for Domain Adaptation in Semantic Segmentation, Frontiers in Computer Science (2022)</a>, Matskevych, Wolny, Pape and Kreshuk</li>
                <li class="paper"><a href="https://doi.org/10.1126/science.abj2949">Profiling cellular diversity in sponges informs animal cell type and nervous system evolution, Science (2021)</a>, Musser, Schippers, Mizzon, Kohn, Pape et al.</li>
                <li class="paper"><a href="https://doi.org/10.1016/j.cell.2021.07.017">Whole-body integration of gene expression and single-cell morphology, Cell (2021)</a>, Vergara, Pape, Meechan et al.</li>
                <li class="paper"><a href="https://doi.org/10.1002/bies.202000257">Microscopy-based assay for semi-quantitative detection of SARS-CoV-2 specific antibdoes in human sera, BioEssays (2021)</a>, Pape, Remme et al.</li>
                <li class="paper"><a href="https://doi.org/10.3389/fcomp.2019.00006">Leveraging domain Knowledge to improve microscopy image segmentation with lifted multicuts, Frontiers in Computer Science (2019)</a>, Pape et al.</li>
                <li class="paper"><a href="https://openaccess.thecvf.com/content_ECCV_2018/html/Steffen_Wolf_The_Mutex_Watershed_ECCV_2018_paper.html">The mutex watershed: efficient, paramter-free image partitioing, ECCV (2018)</a>, Wolf, Pape et al.</li>
                <li class="paper"><a href="https://openaccess.thecvf.com/content_ICCV_2017_workshops/w1/html/Pape_Solving_Large_Multicut_ICCV_2017_paper.html">Solving large multicut problems for connectomics via domain decomposition, ICCV Workshops (2017)</a>, Pape et al.</li>
                <li class="paper"><a href="https://doi.org/10.1038/nmeth.4151">Multicut brings automated neurite segmentation closer to human performance, Nature Methods (2017)</a>, Beier, Pape et al.</li>
                </ul>
            
            </div>
            

            <!-- CV -->
            <div class="col-md-8" style="height: 25vh;">    
                <h2 id="cv">CV</h2>
                
                <ul>
                <li>2010-2013 Bachelor of Science, Physics, Ruprecht-Karls University Heidelberg</li>
                <li>2013-2016 Master of Science, Physics, Ruprecht-Karls University Heidelberg</li>
                <li>2016-2021 PhD, Physics, Ruprecht-Karls University Heidelberg
                    <ul>
                    <li>2017-2018 Visiting scientist, Janelia Research Campus</li>
                    <li>2018-2021 Visiting scientist, EMBL Heidelberg
                    </ul>
                </li>
                <li>2021-2022 PostDoctoral Fellow, EMBL Heidelberg</li>
                <li>2022- Juniorprofessor, Georg-August University Goettingen</li>
                </ul>
            
            </div>

            
        </div>


    </div>
    <!-- /.container -->
    
    <!-- Other people may like it too! -->
    <a style="color:#b5bec9;font-size:0.8em; float:right;" href="https://github.com/mavroudisv/plain-academic">Plain Academic</a> 
    
</body>

</html>
